pocs/utils/messaging.py

```diff
@@ -111,20 +177,24 @@ def send_message(self, channel, message):
         """ Responsible for actually sending message across a channel
 
         Args:
-            channel(str):   Name of channel to send on.
-            message(str):   Message to be sent.
-
+            channel(str):   Name of channel to send on. The name must
+                match name_re.
+            message:   Message to be sent (a string or a dict).
         """
-        assert channel > '', self.logger.warning("Cannot send blank channel")
+        if not isinstance(channel, str):
+            raise ValueError('Channel name must be a string')
+        elif not self.name_re.fullmatch(channel):
+            raise ValueError('Channel name ("{}") is not valid'.format(channel))
 
         if isinstance(message, str):
             message = {
                 'message': message,
-                'timestamp': current_time().isot.replace(
-                    'T',
-                    ' ').split('.')[0]}
-        else:
+                'timestamp': current_time().isot.replace('T', ' ').split('.')[0]
```

Comment:
`current_time(pretty=True)` is the same thing, this code just hasn't been changed since then. Can you update please?